bin/reth/src/node/mod.rs

```diff
@@ -441,6 +449,13 @@ impl<Ext: RethCliExt> Command<Ext> {
         let default_jwt_path = data_dir.jwt_path();
         let jwt_secret = self.rpc.jwt_secret(default_jwt_path)?;
 
+        // Check if `secondary` flag is enabled
+        if self.secondary {
+            self.rpc.auth_port = constants::SECONDARY_AUTH_PORT;
+            self.rpc.http_port = constants::DEFAULT_HTTP_RPC_PORT;
+            self.rpc.ws_port = constants::SECONDARY_WS_RPC_PORT;
+        }
```

Comment:
I want this as a standalone function

Reply:
got it, also I noted I edited wrong the `self.rpc.http_port`. It should be ` = constants::SECONDARY_HTTP_RPC_PORT`.

I can rebase to get rid of clippy errors and make this a standalone function in the meanwhile.